app/lib/tag_manager.rb

```diff
@@ -78,6 +78,8 @@ def uri_for(target)
     case target.object_type
     when :person
       account_url(target)
+    when :note, :comment, :activity
+      unique_tag(target.created_at, target.id, 'Status')
```

Comment:
This is a cheat, of sorts. Relying on `status.created_at` always equaling `stream_entry.created_at` (but they run in the same transaction so it's a safe bet). This saves an extra query